cryptohash-sha256.cabal

```diff
@@ -99,7 +99,7 @@ executable sha256sum
                      , base
                      , bytestring
 
-                     , base16-bytestring >= 0.1.1  && < 0.2
+                     , base16-bytestring >= 0.1.1  && < 1.1
```

Comment:
fwiw, this is an unsound version bound -- unfortunately we don't have a linter for that in cabal yet; no big deal, I'll fix it up while merging